### PR TITLE
C2

### PR DESCRIPTION
**Pull request by christopherchay**
Created super-class Place and sub-class Town

### CODE REVIEW

**Review by celestachay — COMMENTED** [review]
...

File: Classes/Place.py [hunk 1]
```diff
@@ -5,3 +5,69 @@ def __init__(self):
...
+	
+	
+	def GetName(self):
+		return self.Name
+	
+	def GetName(self, name):
+		self.Name = name
+	
+	
+	def GetCanHaveBuildings(self):
```
Comment (cchay) [inline Classes/Place.py]: Rather than "GetCanHaveBuildings()", "CanHaveBuildings():" would read better.

**Comment by EmmanuelChay** [comment]
...

File: Classes/Place.py [hunk 2]
```diff
@@ -5,3 +5,69 @@ def __init__(self):
...
+class Town(Place):
+	def __init__(self):
+		self.CanGoTo = ''
+	
+	def GetCanGoTo(self):
+		return self.CanGoTo
+
+	def SetCanGoTo(self, cangoto):
+		self.CanGoTo = cangoto
+
```
Comment (cchay) [inline Classes/Place.py]: "CanGoTO" is really not a Town attrribute, but the parent's attribute.  And I think a better word/phrase would be appropriate.

File: Classes/Place.py [hunk 3]
```diff
@@ -5,3 +5,69 @@ def __init__(self):
 		self.Description = ''
 		self.Name = ''
 		self.CanHaveBuildings = False
+
```
Comment (cchay) [inline Classes/Place.py]: Lot of inconsistent spacing between defs.

**Review by cchay — CHANGES_REQUESTED** [review]
3 different comments.

File: Classes/Place.py (hunk 2 above)
Comment (christopherchay) [reply Classes/Place.py]: Changed

File: Classes/Place.py (hunk 3 above)
Comment (christopherchay) [reply Classes/Place.py]: The spaces have been fixed as well.

File: Classes/Place.py (hunk 1 above)
Comment (CalebChay) [reply Classes/Place.py]: Besides that, it seems good.

**Review by celestachay — APPROVED** [review]

**Pull request closed without merging**